diff --git a/spec/views/posts/edit.html.tailwindcss_spec.rb b/spec/views/posts/edit.html.tailwindcss_spec.rb
--- a/spec/views/posts/edit.html.tailwindcss_spec.rb
+++ b/spec/views/posts/edit.html.tailwindcss_spec.rb
@@ -2,13 +2,11 @@ require "rails_helper"

 RSpec.describe "posts/edit" do
   let(:post) do
-    Post.create!(
-      title: "MyString",
-      body: "MyText",
-      user: nil
-    )
+    create(:post)
   end

+  let!(:user) { create(:user) }
+
   before do
     assign(:post, post)
   end
@@ -21,7 +19,7 @@ RSpec.describe "posts/edit" do

       assert_select "textarea[name=?]", "post[body]"

-      assert_select "input[name=?]", "post[user_id]"
+      assert_select "select[name=?]", "post[user_id]"
     end
   end
 end
